src/main/java/com/microsoft/sqlserver/jdbc/SQLServerParameterMetaData.java

```diff
@@ -288,9 +288,12 @@ private void checkClosed() throws SQLServerException {
                 String sProc = parseProcIdentifier(st.procedureName);
                 try (SQLServerStatement s = (SQLServerStatement) con.createStatement(ResultSet.TYPE_SCROLL_INSENSITIVE,
                         ResultSet.CONCUR_READ_ONLY);
-                        SQLServerResultSet rsProcedureMeta = s.executeQueryInternal(
-                                con.isKatmaiOrLater() ? "exec sp_sproc_columns_100 " + sProc + ", @ODBCVer=3, @fUsePattern=0"
-                                                      : "exec sp_sproc_columns " + sProc + ", @ODBCVer=3, @fUsePattern=0")) {
+                        SQLServerResultSet rsProcedureMeta = s
```

Comment:
Can we put `s.executeQueryInternal(con.isKatmaiOrLater()` in one line?

Reply:
this is the formatter's doing